src/features/student/components/toggles/TraceAuthenticDeclarationToggle/TraceAuthenticDeclarationToggle.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { AvToggle, type AvToggleProps } from '@avenirs-esr/avenirs-dsav'
+import { useAttrs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+interface TraceAuthenticDeclarationToggleProps extends Omit<AvToggleProps, 'description'> {
```

Comment:
```suggestion
interface TraceAuthenticDeclarationToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'> {
```

Il ne faudrait pas laisser ça en omit du coup ?

Reply:
same https://github.com/avenirs-esr/avenirs-cofolio-client/blob/feat/us-610-update-trace/src/features/student/components/toggles/TraceAuthenticDeclarationToggle/TraceAuthenticDeclarationToggle.vue